File: crud-app/Models/Recipe.cs

using Microsoft.AspNetCore.SignalR;
using Microsoft.Net.Http.Headers;

namespace crud_app;

    public class Recipe
    {
        public int Id { get; set;}
        public required string Recipename { get; set;}
        public string ImageId {get; set;}
        public string Category { get; set;}
        public string Owner {get; set;}
        public required string Description {get; set;}
    }

File: crud-app/ClientApp/src/components/Recipe.js
const Recipe = ({ recipe: { id, recipename, imageId, description, category } }) => {
    return (
        <tr className="recipe" key={id}>
          <h4>
            <p>{recipename}</p>
          </h4>
    
          <p>
            <img src={imageId !== "N/A" ? imageId : "https://via.placeholder.com/400"} alt={recipename} />
          </p>
    
          <td>
            <span>{category}</span>
            <h3>{description}</h3>
          </td>
        </tr>
      );
}

export default Recipe;